python/ray/serve/tests/unit/test_autoscaling_policy.py

```diff
@@ -9,15 +9,22 @@
 
 
 class TestCalculateDesiredNumReplicas:
```

Comment:
AFAICT these are all unit tests, let's make sure there's at least one test that's a full e2e integration test setting the deprecated name

Reply:
Added setting deprecated name in e2e test `test_autoscaling_policy::test_autoscaling_metrics`!